cdc/pkg/config/config_test_data.go

```diff
@@ -81,6 +81,7 @@ const (
     "worker-concurrent": 8,
     "worker-pool-size": 0,
     "region-scan-limit": 40
+	"resolvedts-safe-interval": 3000000000,
```

Comment:
```suggestion
	"resolved-ts-safe-interval": 3s,
```

Reply:
3 * time.Second will convert to 3000000000 after serialization

Reply:
Get it.